packages/frontpage/app/(app)/_components/theme-toggle.tsx

```diff
@@ -18,15 +18,24 @@ export function ThemeToggleMenuGroup() {
       <DropdownMenuLabel className="truncate">Theme</DropdownMenuLabel>
       <DropdownMenuSeparator />
       <DropdownMenuRadioGroup value={theme} onValueChange={setTheme}>
-        <DropdownMenuRadioItem value="light" className="flex gap-1.5">
+        <DropdownMenuRadioItem
+          value="light"
+          className="flex gap-1.5 cursor-pointer"
```

Comment:
We should add this to the base component so that they all share the same style.

Reply:
good point well made